<h1 style="color:black;">Education and Work Experience</h1>
<ul>
  <li>2022 - 2023. Postdoc. Faculty of Engineering, Bar-Ilan University</li>
  <li>2020 - present. Postdoc. Department of Biochemistry and Molecular Genetics, University of Louisville School of Medicine</li>
  <li>2015 - 2020. PhD. Department of Genetics and Genomic Sciences, Icahn School of Medicine at Mount Sinai</li>
  <li>2013 - 2015. Intern, postbac. Department of Genome Sciences, University of Washington</li>
  <li>2014 (summer). Academic contractor. Cancer Signaling Biomarkers & Genetics Group, ImClone Systems (Eli Lilly Company)</li>
  <li>2010 - 2014. Bsc in Biomedical Science (bioinformatics focus), New York University</li>
</ul>

<h1 style="color:black;">Publications</h1>
<h2>2023</h2>
1.  <u>Rodriguez, O. L.</u> et al. [Antibody repertoire gene usage is explained by common genetic variants in the immunoglobulin heavy chain locus](https://www.nature.com/articles/s41467-023-40070-x). <b>Nature Communications</b> 
2.  Pennell, M., <u>Rodriguez, O.L.</u>, Watson, C.T., and Grieff, V.G. [On the evolutionary cause and functional significance of immunoglobulin germline gene variation](https://www.cell.com/trends/immunology/fulltext/S1471-4906(22)00236-8). <b>Trends Immunology</b>. doi:10.1016/j.it.2022.11.001
3.  Gibson, W.S et al. [Characterization of the immunoglobulin lambda chain locus from diverse populations reveals extensive genetic variation](https://www.nature.com/articles/s41435-022-00188-2). <b>Genes & Immunity</b> (2022)
4.  Ford, E. et al. [Flairr-seq: A method for single-molecule resolution of near full-length antibody H chain repertoires](https://journals.aai.org/jimmunol/article/210/10/1607/263586). <b>Journal of Immunology</b>
5. Peres, A. et al. [IGHV allele similarity clustering improves genotype inference from adaptive immune receptor repertoire sequencing data](https://academic.oup.com/nar/article/51/16/e86/7238142) <b>Nucleic Acids Research</b>
6. Li, F. et al. [IgM marks persistent IgG anti-human leukocyte antigen antibodies in highly sensitized heart transplant patients](https://www.sciencedirect.com/science/article/abs/pii/S1053249823020557) <b>Journal of Heart and Lung Transplantation</b>
7. Watson, C. et al. [Looking to the future of antibody genetics: resolving the roles of immunoglobulin diversity in gene regulation, function, and immunity](www.nature.com/articles/s41435-022-00188-2). <b>Genes & Immunity</b> (2022))<b>Genes & Immunity</b>
8.  Ke, H. et al. Human Long Range IGHV Haplotypes Impact Influenza Vaccine Responsiveness. <b>In review</b> (2023)
<h2>2022</h2>
1.  <u>Rodriguez, O. L.</u> et al. [Targeted long-read sequencing facilitates phased diploid assembly and genotyping of the human T cell receptor alpha, delta and beta loci](https://www.cell.com/cell-genomics/pdf/S2666-979X(22)00186-0.pdf). <b>Cell Genomics</b> (2022)
2.  Omer, A. et al. [T Cell Receptor Beta (TRB) Germline Variability is Revealed by Inference From Repertoire Data](https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-01008-4). <b>Genome Medicine</b>. 14, 2 (2022).
3.  Garg, P. et al. [A phenome-wide association study identifies effects of copy number variation of VNTRs and multicopy genes on multiple human traits](https://www.cell.com/ajhg/fulltext/S0002-9297(22)00164-1). <b>Am. J. Hum. Genet.</b> (2022).
4.  Ford, M. et al. [ImmunoTyper-SR: A Novel Computational Approach for Genotyping Immunoglobulin Heavy Chain Variable Genes using Short Read Data](https://www.biorxiv.org/content/10.1101/2022.01.31.478564v1). <b>bioRxiv</b> (2022).
<h2>2021</h2>
1.    <u>Rodriguez, O. L.</u>, Sharp, A., Watson, C. [Limitations of lymphoblastoid cell lines for establishing genetic reference datasets in the immunoglobulin loci.](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0261374) <b>PLOS</b> (2021).
2.    Peng, K. et al. [Diversity in immunogenomics: the value and the challenge](https://www.nature.com/articles/s41592-021-01169-5). <b>Nature Methods</b> (2021)
3.    Garg, P. et al. [Pervasive cis effects of variation in copy number of large tandem repeats on local DNA methylation and gene expression](https://www.sciencedirect.com/science/article/pii/S0002929721001002). <b>Am. J. Hum. Genet</b>. 108, 809–824 (2021).
<h2>2020</h2>
1.    <u>Rodriguez, O. L.</u> et al. [A novel framework for characterizing genomic haplotype diversity in the human immunoglobulin heavy chain locus.](https://www.frontiersin.org/articles/10.3389/fimmu.2020.02136/full) <b>Front. Immunol.</b> (2020), doi:10.3389/fimmu.2020.02136.
2.    Garg, P. et al. [A survey of rare epigenetic variation in 23,116 human genomes identifies disease-relevant epivariations and novel CGG expansions.](https://www.sciencedirect.com/science/article/pii/S0002929720302883) <b>Am. J. Hum. Genet.</b> 107, 654–669 (2020).
3.    Seiden, A. H. et al. [Elucidation of de novo small insertion/deletion biology with parent‐of‐origin phasing.](https://onlinelibrary.wiley.com/doi/full/10.1002/humu.23971) <b>Human Mutation</b> vol. 41 800–806 (2020).
4.    Zook, J. M. et al. [A robust benchmark for detection of germline large deletions and insertions](https://www.nature.com/articles/s41587-020-0538-8) <b> Nature Biotechnol. </b> 38, 1347–1355 (2020).
<h2>2019</h2>
1.    <u>Rodriguez, O. L.</u>, Ritz, A., Sharp, A. J. & Bashir, A. [MsPAC: A tool for haplotype-phased structural variant detection.](https://academic.oup.com/bioinformatics/article/36/3/922/5545544) <b>Bioinformatics</b> (2019) doi:10.1093/bioinformatics/btz618.
2.    Libbrecht, M. W.\*, <u>Rodriguez, O. L.</u>\* (\*contributed equally). et al. [A unified encyclopedia of human functional DNA elements through fully automated annotation of 164 human cell types](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1784-2) <b>Genome Biology</b>. 20, 180 (2019).
3.    Cirelli, K. M. et al. [Slow Delivery Immunization Enhances HIV Neutralizing Antibody and Germinal Center Responses via Modulation of Immunodominance.](https://www.sciencedirect.com/science/article/pii/S0092867419303988?via%3Dihub) <b>Cell</b> 0, (2019).
4.    Chaisson, M. J. P. et al. [Multi-platform discovery of haplotype-resolved structural variation in human genomes.](https://www.nature.com/articles/s41467-018-08148-z) <b>Nature Commun.</b> 10, 1784 (2019).
<h2>2016</h2>
5.     Moyer, E. et al. [MetaNetVar: Pipeline for applying network analysis tools for genomic variants analysis.](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4857755/) <b>F1000Res</b>. 5, 674 (2016).


<h1 style="color:black;">Awards</h1>
1. 2022-2024 Zuckerman Postdoctoral Scholar, Zuckerman STEM Leadership Program, Zuckerman Institute
1. 2020-2022 Human Genetics Scholar, American Society of Human Genetics
2. 2020 Charles J. Epstein Trainee Awards for Excellence in Human Genetics Research semifinalist, American Society of Human Genetics
3. 2018-2020 F31 Predoctoral Ruth L. Kirschstein National Research Service Award, National Institute of Neurological Disorders and Stroke, National Institute of Health
5. 2017 Mindich Child Health and Development Institute Trainee Pilot Program Grant
6. 2016 American Society of Human Genetics Reviewer's Choice Abstract (abstract scored in top 10% of poster abstracts) and featured in Poster Walk. 
7. Society for the Advancement of Chicanos/Hispanics and Native Americans 2013 National Conference Travel Scholarship
8. University of Washington Genomic Outreach for Minorities Project Travel Scholarship, 2013
9. University of Washington Genomic Outreach for Minorities Project Travel Scholarship, 2014
10. Boston University Bioinformatics Diversity Travel Fellowship, 2014

<h1 style="color:black;">Software</h1>
1. [Pipeline for diploid assembly and structural variant detection](https://github.com/oscarlr/MsPAC)
2. [Framework to resolve the immunoglobulin heavy chain locus](https://github.com/oscarlr/IGenotyper)
